src/compositions/useLspFactory.ts

```diff
@@ -51,7 +57,10 @@ export function useLspFactory(): {
   if (!hasExtension) {
     throw new Error("Extension not installed");
   }
-  lspFactory = new LSPFactory(window.ethereum);
+  setupLSPFactory();
+
+  // lspFactory = new LSPFactory(window.ethereum);
```

Comment:
Do we need this?

Reply:
Wanted to confirm with Andrew, before getting rid of it, since he wrote the composable, but I doubt we'd be needing it.